packages/core/src/odata/v4/request-builder/create-request-builder.ts

```diff
@@ -87,7 +87,7 @@ export class CreateRequestBuilder<EntityT extends Entity>
       .then(request => request.execute())
       .then(response =>
         deserializeEntity(
-          extractData(response),
+          getSingleResult(response.data),
```

Comment:
Is it always `.data` or should we move also this extraction part to the getter methods?

Reply:
It is not always data. In batch for example we first have to parse the reponse.data, to get this.